packages/hadron-app-registry/src/register-plugin.tsx

```diff
@@ -197,6 +261,8 @@ function useHadronPluginActivate<T, S extends Record<string, () => unknown>>(
     })
   ) as Services<S>;
 
+  serviceLocationInProgress = false;
```

Comment:
Should this be reset in a `finally` block so that this behaves well if calling the locator fails above?